### PR TITLE
feat(header): update design

### LINKED ISSUE #248: Header for account(s)/login/signup
The header for account(s)/signup/login should match the design in Figma. Note, that they are different. Currently, they use the same styles, and it doesn't look good enough.

### PR DESCRIPTION
**Pull request by andreybakanovsky**
It resolves #248

### CODE REVIEW

**Comment by andreybakanovsky** [comment]
image

**Comment by andreybakanovsky** [comment]
image

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -64,14 +64,14 @@
   </head>
 
   <body class=<%= show_application_background? ? 'with-background' : ''%>>
-    <div class="container hero is-fullheight">
-      <nav class="navbar is-spaced has-shadow is-navbar-budgeting-kid-color"
-           role="navigation"
-           aria-label="main navigation">
+    <nav class="navbar wrapper is-navbar-budgeting-kid-color <%= navbar_hide? ? 'hidden' : ''%>"
```
Comment (VladislavSokov) [inline app/views/layouts/application.html.erb:67]: add please `is-radiusless` class

Comment (andreybakanovsky) [reply app/views/layouts/application.html.erb:67]: @VladislavSokov, could you please clarify your statement, for which element and why?

Comment (andreybakanovsky) [reply app/views/layouts/application.html.erb:67]: It's a feature. Ok, if you'd like.

**Comment by andreybakanovsky** [comment]
I haven't seen the issue(, I need to change it, it's at higher display resolutions :
image

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -11,6 +11,13 @@ def ps
 
   private
 
+  helper_method def navbar_hide?
```
Comment (ka8725) [inline app/controllers/application_controller.rb:14]: WDYT if we just created separate layouts and tell these controllers to use them?

**Comment by andreybakanovsky** [comment]
scale 75%


image
image
image

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (andreybakanovsky) [reply app/controllers/application_controller.rb]: ! Thank you

File: app/views/layouts/_head.html.erb [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    description = content_for?(:description) ? yield(:description) : <<~description
+        Empower Kids with Financial Wisdom. Teach, track, and save effortlessly with our app.
+        Start their journey to financial literacy now!
+        description
+%>
+
+<link rel="canonical" href="<%= current_url %>">
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta name="title" content="<%= title %>">
```
Comment (ka8725) [inline app/views/layouts/_head.html.erb:14]:
```suggestion
```

File: app/views/layouts/initial_application.html.erb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <%= render 'layouts/head'%>
+  </head>
+
+  <body>
+      <%= render 'layouts/header/initial' %>
```
Comment (ka8725) [inline app/views/layouts/initial_application.html.erb:8]:
```suggestion
    <%= render 'layouts/header/initial' %>
```

File: app/views/layouts/initial_application.html.erb [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    <div class="container hero is-fullheight bottom-wrapper " >
+      <section class="section is-fullheight py-0 vcentered-columns">
+        <%= yield %>
+      </section>
+      <%= render 'layouts/footer/footer'%>
```
Comment (ka8725) [inline app/views/layouts/initial_application.html.erb:13]:
```suggestion
      <%= render 'layouts/footer/footer' %>
```

File: app/views/layouts/header/_main.html.erb [hunk 6]
```diff
@@ -0,0 +1,83 @@
+<nav class="navbar wrapper is-navbar-budgeting-kid-color"
+      role="navigation"
+      aria-label="main navigation">
+  <div class="container px-4 container-buttom-border">
+    <div class="navbar-brand">
+      <%= link_to root_path, class: "navbar-item" do %>
+        <%= image_tag 'logo.svg', size: "44"%>
+        <H2 class='budgeting-kid-logo-font px-4'> BUDGETINGkid </H2>
```
Comment (ka8725) [inline app/views/layouts/header/_main.html.erb:8]:
```suggestion
        <h2 class='budgeting-kid-logo-font px-4'> BUDGETINGkid </h2>
```

File: app/views/layouts/header/_main.html.erb [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+      </a>
+    </div>
+    <div id="navbarMenu" class="navbar-menu">
+      <div class="navbar-end">
+        <div class="navbar-item pb-4">
+          <div class="buttons">
+            <% if user_signed_in? %>
+              <div class="navbar-item has-dropdown is-hoverable">
+                <a class="button is-rounded for-desktop p-0 m-0" style="width: 72px; height: 72px;" >
+                  <img src=<%=asset_path(avatar_for(current_user))%> class="rounded-avatar small">
```
Comment (ka8725) [inline app/views/layouts/header/_main.html.erb:23]:
```suggestion
                  <img src=<%= asset_path(avatar_for(current_user)) %> class="rounded-avatar small">
```

File: app/views/layouts/header/_initial.html.erb [hunk 8]
```diff
@@ -0,0 +1,11 @@
+<nav class="navbar wrapper is-navbar-budgeting-kid-color"
+      role="navigation"
+      aria-label="main navigation">
+  <div class="container px-4">
+    <div class="navbar-brand">
+      <%= link_to root_path, class: "navbar-item" do %>
+        <%= image_tag 'logo.svg', size: "44"%>
+        <H2 class='budgeting-kid-logo-font px-4'> BUDGETINGkid </H2>
```
Comment (ka8725) [inline app/views/layouts/header/_initial.html.erb:8]:
```suggestion
        <h2 class='budgeting-kid-logo-font px-4'> BUDGETINGkid </h2>
```

File: app/views/layouts/_head.html.erb [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+<%= favicon_link_tag 'favicons/favicon.ico'%>
+<%= favicon_link_tag 'favicons/favicon.svg'%>
+
+<% %w(16 32 48 96 144 192 256 384 512 70 150 310).each do |s| %>
+    <%= favicon_link_tag "favicons/favicon_#{s}.png",sizes: "#{s}x#{s}" %>
+<% end %>
+
+<% %w(32 76 120 152 167 180).each do |s| %>
+    <%= favicon_link_tag "favicons/favicon_#{s}.png",sizes: "#{s}x#{s}", rel: 'apple-touch-icon', type: 'image/png' %>
```
Comment (ka8725) [inline app/views/layouts/_head.html.erb:47]:
```suggestion
  <%= favicon_link_tag "favicons/favicon_#{s}.png",sizes: "#{s}x#{s}", rel: 'apple-touch-icon', type: 'image/png' %>
```

File: app/views/layouts/_head.html.erb [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+<!-- facebook -->
+
+<%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+<%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+
+<%= favicon_link_tag 'favicons/favicon.ico'%>
+<%= favicon_link_tag 'favicons/favicon.svg'%>
+
+<% %w(16 32 48 96 144 192 256 384 512 70 150 310).each do |s| %>
+    <%= favicon_link_tag "favicons/favicon_#{s}.png",sizes: "#{s}x#{s}" %>
```
Comment (ka8725) [inline app/views/layouts/_head.html.erb:43]:
```suggestion
  <%= favicon_link_tag "favicons/favicon_#{s}.png",sizes: "#{s}x#{s}" %>
```

File: app/views/layouts/_head.html.erb [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+<% if ENV['GOOGLE_TAG_ID'] %>
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=<%= ENV['GOOGLE_TAG_ID'] %>"></script>
+    <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', '<%= ENV['GOOGLE_TAG_ID'] %>');
+    </script>
```
Comment (ka8725) [inline app/views/layouts/_head.html.erb:59]: use 2 spaces indentation

**Review by ka8725 — APPROVED** [review]

**Pull request merged**